account_invoice_view_payment/wizards/__init__.py

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
nitpicking (optional):
missing lincense
 License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).

Reply:
IMHO, this is not necessary in init files